### DISCUSSION TITLE
Looking for maintainers!

### DISCUSSION THREAD

**Discussion by mmazzarolo** [react-native-modal/react-native-modal#598] [Announcements]
# 👉We're looking for maintainers! 

If you're interested in helping out, please start contributing and then post here :)

**Comment by caifel** [comment]
Hi @mmazzarolo I can help. First community library in which I would work.

**Comment by ancyrweb** [comment]
@GenDecoder i've sorted the issues today and added labels. There are some good first issues for you to get started with. Ping me if you need help :)

**Comment by Jmzp** [comment]
Hello @mmazzarolo, I can help, like @GenDecoder this is the first community library that I would work on.

**Comment by binbandit** [comment]
@mmazzarolo I'm more than happy to help.

I have released components for other projects before and actively maintain them too.

**Comment by mmazzarolo** [comment]
👍 There are some good first issues (labelled) to get started with. Ping us if you need any help!

**Comment by peterlogo** [comment]
Hi @mmazzarolo this will be my first community library to be part of. I would love to be part of the maintainers as well.
I know it will be a good learning experience.

**Comment by BatDroid** [comment]
Hi @mmazzarolo , I will start my contribution as soon as possible. hope to become a contributor here :)

**Comment by yjb94** [comment]
Hi @mmazzarolo, wish to be part of maintainers as well. plz check my PR :)

**Comment by kaushikdas0** [comment]
Hi @mmazzarolo i would like to help as well. This will be my first community build. Thanks

**Comment by xDemon200** [comment]
Hi @mmazzarolo I can help. First community library in which I would work.

**Comment by ansarikhurshid786** [comment]
Hi i can help

**Comment by Amruthalingam** [comment]
Hi my first one too, would like to contribute to our community

**Comment by sanathnair09** [comment]
Hey @mmazzarolo I would like to help contribute. Like many others this would be my first contribution!

**Comment by mayank-budhiraja** [comment]
Hey @mmazzarolo I would like to help. Please, let me know if you have anything for me.

**Comment by NathanBeesley** [comment]
I'm looking to help out the react-native community. Are you still looking for maintainers? Either way I'll check out some issues.

**Comment by mmazzarolo** [comment]
Thank you @NathanBeesley :) (and everyone else who commented). 
Feel free to tackle existing issues with questions/answers/PRs and report back here if needed. After some contributions I'll help with permissions/etc...

**Comment by NathanBeesley** [comment]
I've been looking through some of the issues and it seems like there are a lot of outstanding questions. I'll try to go through them and help resolve issues and provide suggestions.

**Comment by mmazzarolo** [comment]
Thank you @NathanBeesley , I added you as a collaborator with Triage permission (for now) — you should be able to manage issues/pull-requests and add tags/close duplicates. If you need any additional "quick" help, please send me an email 👍 (I'm still keeping an eye on the thread though)

**Comment by NathanBeesley** [comment]
I was going to start by tagging some issues that do not provide enough repro steps to question need further information but I'm unable to. Not sure if Triage permission gets those permissions? Just trying to isolate, clean up some of them in hopes of getting responses from the creator.

**Comment by mmazzarolo** [comment]
@NathanBeesley I think you need to accept the invite
![image](https://user-images.githubusercontent.com/9536354/119514524-61871c80-bd75-11eb-9d32-45df2c415cbf.png)

**Comment by NathanBeesley** [comment]
You are right. I didn't even see the notification. I'm blind.

**Comment by betomoedano** [comment]
hello, I would like to help.

**Comment by hassanAwanAlvi** [comment]
Hi, I would like to help...

**Comment by BilguunTs** [comment]
Ready to go

**Comment by souzaluiz** [comment]
Hi, I would like to help.

**Comment by os2mem** [comment]
I would like to help :D

**Comment by TheRogue76** [comment]
Hey @mmazzarolo. I've been using your library for a couple of years now. I would be glad to help❤️

**Reply by mmazzarolo** [reply]
Thank you! 
Feel free to join any open issue to answer, add more context, or ask for questions :)

**Comment by mnghn07** [comment]
Hi @mmazzarolo , I really like this library. Been using it for a while now. Would be glad to help to contribute! 💪

**Comment by ansarikhurshid786** [comment]
I will also try my best

**Comment by abdallaemadeldin** [comment]
Hey i like to contribute with you guys

**Comment by peterlogo** [comment]
Hey I will like to contribute as well

**Comment by anonymous887** [comment]
I am Interested

**Comment by sevysevy** [comment]
hello i will be happy if i can give some help as well

**Comment by os2mem** [comment]
Hi I would like to help!

**Comment by jsantanders** [comment]
Hi I would like to help!

**Comment by ifeanyi-ugwu** [comment]
Hi @mmazzarolo I am using React native currently for my project and would love to contribute to this community library.

**Comment by moniquealtero** [comment]
Any updates?

**Comment by Ahmedhamed77** [comment]
hi, I would like to help !

**Comment by Sacre-MBIKU-WAKANE** [comment]
Hello @mmazzarolo  ! I'll be happy to help, and this will be my first participation in an opensource project.

**Comment by RayanAbid** [comment]
Would love to start helping out. I'll check the issues if any.

**Comment by hookFang** [comment]
Hi Team, would love to help you out with some tickets. Let me know if there is something I can start on. I have contributed and worked on several react-native and react projects.

**Comment by AhteshamKhanCoder** [comment]
@mmazzarolo  I would like to work for the maintenance

**Comment by Leadola1** [comment]
I would like to help
